src/ResourceManager/Maps/Maps.Test/ScenarioTests/MapsAccountTests.ps1

```diff
@@ -0,0 +1,277 @@
+﻿# ----------------------------------------------------------------------------------
+#
+# Copyright Microsoft Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+
+<#
+.SYNOPSIS
+Test New-AzureRmMapsAccount
+#>
+function Test-NewAzureRmMapsAccount
+{
+    # Setup
+    $rgname = Get-MapsManagementTestResourceName;
+
+    try
+    {
+        # Test
+        $accountname = 'ps-' + $rgname;
+        $skuname = 'S0';
+        $location = 'West US';
+
+        New-AzureRmResourceGroup -Name $rgname -Location $location;
+
+        $createdAccount = New-AzureRmMapsAccount -ResourceGroupName $rgname -Name $accountname -SkuName $skuname -Force;
+        Assert-NotNull $createdAccount;
+        # Call create again, expect to get the same account
+        $createdAccountAgain = New-AzureRmMapsAccount -ResourceGroupName $rgname -Name $accountname -SkuName $skuname -Force;
+        Assert-NotNull $createdAccountAgain
+        Assert-AreEqual $createdAccount.Id $createdAccountAgain.Id;
+        Assert-AreEqual $createdAccount.ResourceGroupName $createdAccountAgain.ResourceGroupName;
+        Assert-AreEqual $createdAccount.Name $createdAccountAgain.Name;
+        Assert-AreEqual $createdAccount.Location $createdAccountAgain.Location;
+        Assert-AreEqual $createdAccount.Sku.Name $createdAccountAgain.Sku.Name;
+        
+        Retry-IfException { Remove-AzureRmMapsAccount -ResourceGroupName $rgname -Name $accountname -Confirm:$false; }
```

Comment:
Please add a check here to ensure the account was deleted (make sure get returns nothing)

Reply:
Looks like the check is in under `function Test-RemoveAzureRmMapsAccount` at [line 73 & 83](https://github.com/jp94/azure-powershell/blob/c348a3e83ff522e26ed593d6cf9506c6fc219454/src/ResourceManager/Maps/Maps.Test/ScenarioTests/MapsAccountTests.ps1#L73). I think this check is good, but I can also add to all other test cases if needed.